build-exe.mac.sh

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash
+# vi: ts=4 sw=4 et syntax=sh :
+
+cd "$(dirname "${BASH_SOURCE[0]}")" || exit 1
+
+
+NAME='import-mailbox-to-gmail'
+BUILD_DIR="./build"
+
+# Regenerate with, e.g. $ pwgen 16 1
+SIGNING_KEY='jouNae6aifeideeL'
+
+_uname="$(uname)"
+case "${_uname}" in
+    [dD]arwin)
+        ;;
+    *)
+        die "OS not supported: ${_uname}"
+        ;;
+esac
+
+TOOL_pyinstaller="$(which pyinstaller 2>/dev/null)"
+TOOL_pyi_makespec="$(which pyi-makespec 2>/dev/null)"
+
+if [[ -z "${TOOL_pyinstaller}" || -z "${TOOL_pyi_makespec}" ]]; then
+    echo "Missing required tool: pyinstaller" >&2
```

Comment:
They are both part of PyInstaller, so I believe declaring PyInstaller missing is the best option.
We can skip the explicit check for ``$TOOL_pyi_makespec`` not being empty, but since I need anyway to spend time to resolve it, I preferred to keep it.